blueprints/app/files/app/router.__ext__

```diff
@@ -6,4 +6,4 @@ export default class Router extends EmberRouter {
   rootURL = config.rootURL;
 }
 
-Router.map(function () {});
+Router.map(function () {});<% if (typescript) { %> // eslint-disable-line @typescript-eslint/no-empty-function<% } %>
```

Comment:
I'm surprised `no-empty-function` is part of the `@typescript-eslint` recommended ruleset but not the regular `eslint:recommended` one. (But that's neither here nor there 😄)

Would changing this to something like the snippet below appease the rule without us needing to explicitly disable it?

```ts
Router.map(function () {
  // Add route declarations here
});
```

Reply:
> Would changing this to something like the snippet below appease the rule without us needing to explicitly disable it?

Yes, it would!

I just wanted to try out what happens with the comment when you add a route by `ember g route`, only to find out that it actually does not work at all with a `router.ts` file! Is this known?

Not sure if https://github.com/ember-cli/ember-router-generator would actually support parsing TS, but the error is thrown even before that lib comes into play, in the ember-source blueprint [here](https://github.com/emberjs/ember.js/blob/master/blueprints/route/index.js#L165-L166), as it only checks for a `router.js`.

Reply:
Yep, that's [a known issue](https://github.com/typed-ember/ember-cli-typescript/issues/165) (Which we should definitely fix now that TS support is official! Though obviously not a hard blocker for this PR in particular)

Reply:
Now using the suggested comment in the (otherwise still empty) function body!